### PR TITLE
Config updates

### PR DESCRIPTION
**Pull request by Crossedfall**
_No CL needed._

Removes the example config directory and adds in our most current config files and title screens. Music is omitted for licensing reasons.

Config changes are no longer ignored by git.

### CODE REVIEW

**Comment by Crossedfall** [comment]
Travis had an oopsie. Can someone restart the failed job please? Just gotta hit this icon next to the failed job. <3

image

**Comment by qwertyquerty** [comment]
rerunning rn

**Comment by qwertyquerty** [comment]
We don't use config/BeeStation-Glitch.dmi anymore iirc so that can be removed

File: .gitignore [hunk 1]
```diff
@@ -210,5 +210,8 @@ tools/MapAtmosFixer/MapAtmosFixer/bin/*
 /_maps/**/backup/
 /_maps/templates.dm
 
-# config
-config/*
```
Comment (qwertyquerty) [inline .gitignore:214]: We aren't going to ignore whitelist.txt admins.txt comms.txt and dbconfig.txt?

Comment (Crossedfall) [reply .gitignore:214]: They're ignored on the server side of things. So, changes merged in for those files are not live unless I or Cthulhu manually apply them.

Comment (qwertyquerty) [reply .gitignore:214]: Yes but some players will prefer to keep their name in as an admin for testing

Comment (Crossedfall) [reply .gitignore:214]: The local user is always admin. There is no reason to touch those files during testing.

Comment (qwertyquerty) [reply .gitignore:214]: I've never had admin as the local user what

**Pull request merged**